src/vhpi/vhpi-util.c

```diff
@@ -298,6 +298,14 @@ vhpiPhysT vhpi_phys_from_native(int64_t value)
    return result;
 }
 
+vhpiIntT vhpi_int_from_native(int64_t value)
+{
+   if (value > INT32_MAX)
+      fatal_trace("value %" PRId64 "is too large for vhpiIntT", value);
```

Comment:
Need a `#include <inttypes.h>` on Windows/Mac (apparently it gets included transitively from somewhere else on Linux).

Reply:
Also the comparison ought to be `value < INT32_MIN || value > INT32_MAX` to catch too large negative values.